lib/StaticNarrative/exporter/data_exporter.py

```diff
@@ -1,25 +1,24 @@
 """Fetches data to be exported as part of the Static Narrative creation."""
 
 import json
-import os
+from pathlib import Path
 from typing import Any
 
 from installed_clients.WorkspaceClient import Workspace
 
 from StaticNarrative.exporter.dynamic_service_client import DynamicServiceClient
 from StaticNarrative.exporter.objects_with_sets import ObjectsWithSets
-
-from .processor_util import get_data_icon
+from StaticNarrative.exporter.processor_util import get_data_icon
 
 IGNORED_TYPES = ["KBaseNarrative.Narrative"]
 
 
 def export_narrative_data(
     ws_client: Workspace,
     wsid: int,
-    output_dir: str,
-    service_wizard_url: str,
     token: str,
+    output_dir: str | Path,
+    set_api_client: DynamicServiceClient | None = None,
```

Comment:
pass the client in instead of passing in the URL (will be easier for testing)

Reply:
Always easier, whenever possible!